modules/core/src/test/java/org/apache/ignite/internal/encryption/CacheGroupReencryptionTest.java

```diff
@@ -631,6 +631,38 @@ public void testReencryptionOnUnstableTopology() throws Exception {
         checkGroupKey(CU.cacheId(cache2), INITIAL_KEY_ID + 1, MAX_AWAIT_MILLIS);
     }
 
+    /**
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testDeactivation() throws Exception {
+        pageScanRate = 1;
+
+        T2<IgniteEx, IgniteEx> nodes = startTestGrids(true);
+
+        IgniteEx node0 = nodes.get1();
+        IgniteEx node1 = nodes.get2();
+
+        createEncryptedCache(node0, node1, cacheName(), null);
+
+        loadData(100_000);
+
+        node0.encryption().changeCacheGroupKey(Collections.singleton(cacheName())).get();
+
+        assertTrue(isReencryptionInProgress(Collections.singleton(cacheName())));
+
+        node0.cluster().state(ClusterState.INACTIVE);
+
```

Comment:
```
        stopGrid(GRID_1);
        startGrid(GRID_1);
```
Reproduces the problem I've mentioned above

Reply:
Added this case and fix for it.
(from my point of view, the name of the ``isReencryptionInProgress`` method in the test is not entirely correct - this method checks that re-encryption is required, but it doesn’t say whether the re-encryption task is running (), I can’t remember why this name was chosen)